java.lang.NoClassDefFoundError when importing project in IntelliJ

## steps

We have a project running (happily) with sbt 1.4.7. After updating to 1.4.8, we are getting an exception in IntelliJ when reloading the project.

## problem

When reloading the project in the IDE, we are getting the following exception in "sbt shell" window:
```
[snipped]
java.lang.NoClassDefFoundError: org/fusesource/jansi/AnsiOutputStream
	at jline.internal.Ansi.stripAnsi(Ansi.java:29)
	at jline.console.ConsoleReader.setPrompt(ConsoleReader.java:528)
	at jline.console.ConsoleReader.readLine(ConsoleReader.java:2454)
	at jline.console.ConsoleReader.readLine(ConsoleReader.java:2407)
	at sbt.internal.util.Terminal$$anon$4.super$readLine(Terminal.scala:786)
	at sbt.internal.util.Terminal$$anon$4.$anonfun$readLine$1(Terminal.scala:786)
	at sbt.internal.util.Terminal.withRawInput(Terminal.scala:145)
	at sbt.internal.util.Terminal.withRawInput$(Terminal.scala:143)
	at sbt.internal.util.Terminal$TerminalImpl.withRawInput(Terminal.scala:887)
	at sbt.internal.util.Terminal$$anon$4.readLine(Terminal.scala:786)
	at jline.console.ConsoleReader.readLine(ConsoleReader.java:2395)
	at sbt.internal.util.Terminal$$anon$4.super$readLine(Terminal.scala:787)
	at sbt.internal.util.Terminal$$anon$4.$anonfun$readLine$2(Terminal.scala:787)
	at sbt.internal.util.Terminal.withRawInput(Terminal.scala:145)
	at sbt.internal.util.Terminal.withRawInput$(Terminal.scala:143)
	at sbt.internal.util.Terminal$TerminalImpl.withRawInput(Terminal.scala:887)
	at sbt.internal.util.Terminal$$anon$4.readLine(Terminal.scala:787)
	at sbt.internal.util.JLine.readLineDirectRaw(LineReader.scala:209)
	at sbt.internal.util.JLine.$anonfun$readLineDirect$2(LineReader.scala:200)
	at sbt.internal.util.Signals0.withHandler(Signal.scala:92)
	at sbt.internal.util.Signals$.withHandler(Signal.scala:20)
	at sbt.internal.util.JLine.readLineDirect(LineReader.scala:200)
	at sbt.internal.util.JLine.readLineWithHistory(LineReader.scala:194)
	at sbt.internal.util.JLine.unsynchronizedReadLine(LineReader.scala:185)
	at sbt.internal.util.JLine.readLine(LineReader.scala:177)
	at org.jetbrains.sbt.pluginCommands$.$anonfun$ideaShell$1(pluginCommands.scala:22)
	at sbt.Command$.$anonfun$command$2(Command.scala:97)
	at sbt.Command$.process(Command.scala:189)
	at sbt.MainLoop$.$anonfun$processCommand$5(MainLoop.scala:243)
	at scala.Option.getOrElse(Option.scala:189)
	at sbt.MainLoop$.process$1(MainLoop.scala:243)
	at sbt.MainLoop$.processCommand(MainLoop.scala:274)
	at sbt.MainLoop$.$anonfun$next$5(MainLoop.scala:163)
	at sbt.State$StateOpsImpl$.runCmd$1(State.scala:289)
	at sbt.State$StateOpsImpl$.process$extension(State.scala:325)
	at sbt.MainLoop$.$anonfun$next$4(MainLoop.scala:163)
	at sbt.internal.util.ErrorHandling$.wideConvert(ErrorHandling.scala:23)
	at sbt.MainLoop$.next(MainLoop.scala:163)
	at sbt.MainLoop$.run(MainLoop.scala:144)
	at sbt.MainLoop$.$anonfun$runWithNewLog$1(MainLoop.scala:119)
	at sbt.io.Using.apply(Using.scala:27)
	at sbt.MainLoop$.runWithNewLog(MainLoop.scala:112)
	at sbt.MainLoop$.runAndClearLast(MainLoop.scala:66)
	at sbt.MainLoop$.runLoggedLoop(MainLoop.scala:51)
	at sbt.MainLoop$.runLogged(MainLoop.scala:42)
	at sbt.StandardMain$.runManaged(Main.scala:193)
	at sbt.xMain$.$anonfun$run$8(Main.scala:102)
	at scala.util.DynamicVariable.withValue(DynamicVariable.scala:62)
	at scala.Console$.withIn(Console.scala:230)
	at sbt.internal.util.Terminal$.withIn(Terminal.scala:560)
	at sbt.internal.util.Terminal$.$anonfun$withStreams$1(Terminal.scala:350)
	at scala.util.DynamicVariable.withValue(DynamicVariable.scala:62)
	at scala.Console$.withOut(Console.scala:167)
	at sbt.internal.util.Terminal$.$anonfun$withOut$2(Terminal.scala:550)
	at scala.util.DynamicVariable.withValue(DynamicVariable.scala:62)
	at scala.Console$.withErr(Console.scala:196)
	at sbt.internal.util.Terminal$.withOut(Terminal.scala:550)
	at sbt.internal.util.Terminal$.withStreams(Terminal.scala:350)
	at sbt.xMain$.run(Main.scala:86)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at sbt.internal.XMainConfiguration.run(XMainConfiguration.scala:83)
	at sbt.xMain.run(Main.scala:46)
	at xsbt.boot.Launch$$anonfun$run$1.apply(Launch.scala:111)
	at xsbt.boot.Launch$.withContextLoader(Launch.scala:131)
	at xsbt.boot.Launch$.run(Launch.scala:111)
	at xsbt.boot.Launch$$anonfun$apply$1.apply(Launch.scala:37)
	at xsbt.boot.Launch$.launch(Launch.scala:120)
	at xsbt.boot.Launch$.apply(Launch.scala:20)
	at xsbt.boot.Boot$.runImpl(Boot.scala:56)
	at xsbt.boot.Boot$.main(Boot.scala:18)
	at xsbt.boot.Boot.main(Boot.scala)
Caused by: java.lang.ClassNotFoundException: org.fusesource.jansi.AnsiOutputStream
	at java.base/java.net.URLClassLoader.findClass(URLClassLoader.java:471)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:589)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
	... 74 more
[error] java.lang.NoClassDefFoundError: org/fusesource/jansi/AnsiOutputStream
[error] Use 'last' for the full log.

Process finished with exit code 1
```

## expectation

It should reload as expected.

## notes

--- a/project/Dependencies.scala
+++ b/project/Dependencies.scala
@@ -84,14 +84,16 @@ object Dependencies {
   val sjsonNewScalaJson = sjsonNew("sjson-new-scalajson")
   val sjsonNewMurmurhash = sjsonNew("sjson-new-murmurhash")
 
-  val jline = "org.scala-sbt.jline" % "jline" % "2.14.7-sbt-42b717d4418374417765c7651dca69b1b75d8b84"
+  // JLine 3 version must be coordinated together with JAnsi version
+  // and the JLine 2 fork version, which uses the same JAnsi
+  val jline = "org.scala-sbt.jline" % "jline" % "2.14.7-sbt-a1b0ffbb8f64bb820f4f84a0c07a0c0964507493"
   val jline3Version = "3.19.0"
   val jline3Terminal = "org.jline" % "jline-terminal" % jline3Version
   val jline3Jansi = "org.jline" % "jline-terminal-jansi" % jline3Version
   val jline3JNA = "org.jline" % "jline-terminal-jna" % jline3Version
   val jline3Reader = "org.jline" % "jline-reader" % jline3Version
   val jline3Builtins = "org.jline" % "jline-builtins" % jline3Version
-  val jansi = "org.fusesource.jansi" % "jansi" % "2.0.1"
+  val jansi = "org.fusesource.jansi" % "jansi" % "2.1.0"
   val scalatest = "org.scalatest" %% "scalatest" % "3.0.8"
   val scalacheck = "org.scalacheck" %% "scalacheck" % "1.14.0"
   val junit = "junit" % "junit" % "4.13.1"

